### PR TITLE
Encrypted shared volume description.

### PR DESCRIPTION
**Pull request by talakad**

### CODE REVIEW

File: manage/encrypted-volumes.md [hunk 1]
```diff
@@ -63,3 +63,47 @@ Volume successfully created: 374663852714325215
...
 __Important: Make sure secret `key1` exists in the secret endpoint__
+
+## Encrypted Shared Volumes
+
+Encrypted shared volume allows access from multiple nodes to the same
+encrypted volume.
+
+Shared flag can be set while creating the encrypted volume using `--shared`
+It can also be enabled or disabled during run-time using `--shared on/off`.
+Volume must be in detached to state to enable shared flag on during run-time.
```
Comment (piyush-nimbalkar) [inline manage/encrypted-volumes.md]: must be in detached state to toggle shared flag during run-time

File: manage/encrypted-volumes.md [hunk 2]
```diff
@@ -63,3 +63,47 @@ Volume successfully created: 374663852714325215
...
+
+All nodes in the cluster must be authenticated to access secret store for
```
Comment (piyush-nimbalkar) [inline manage/encrypted-volumes.md]: Just say "Portworx cluster must be authenticated to ..."

File: manage/encrypted-volumes.md [hunk 3]
```diff
@@ -63,3 +63,47 @@ Volume successfully created: 374663852714325215
...
+the encryption keys.
+Both cluster wide and per volume secrets are supported. For example, using
+cluster wide secret key;
```
Comment (piyush-nimbalkar) [inline manage/encrypted-volumes.md]: ':' in the end

File: manage/encrypted-volumes.md [hunk 4]
```diff
@@ -63,3 +63,47 @@ Volume successfully created: 374663852714325215
...
+
+```
+#pxctl volume create --shared --secure --size 10 encrypted_volume
```
Comment (piyush-nimbalkar) [inline manage/encrypted-volumes.md]: space after #

File: manage/encrypted-volumes.md [hunk 5]
```diff
@@ -63,3 +63,47 @@ Volume successfully created: 374663852714325215
...
+Bytes Read      	 :  0
+Writes          	 :  0
+Writes MS       	 :  0
+Bytes Written   	 :  0
+IOs in progress 	 :  0
+Bytes used      	 :  131 MiB
+Replica sets on nodes:
+	Set 0
+		Node 		 : 70.0.18.11 (Pool 0)
+eplication Status	 :  Detached
```
Comment (piyush-nimbalkar) [inline manage/encrypted-volumes.md]: line is trimmed (Replication Status)
and extra # on line 108

File: manage/encrypted-volumes.md (hunk 1 above)
Comment (talakad) [reply manage/encrypted-volumes.md]: done.

File: manage/encrypted-volumes.md (hunk 2 above)
Comment (talakad) [reply manage/encrypted-volumes.md]: done

File: manage/encrypted-volumes.md (hunk 3 above)
Comment (talakad) [reply manage/encrypted-volumes.md]: done

File: manage/encrypted-volumes.md (hunk 4 above)
Comment (talakad) [reply manage/encrypted-volumes.md]: done

File: manage/encrypted-volumes.md (hunk 5 above)
Comment (talakad) [reply manage/encrypted-volumes.md]: done

File: manage/encrypted-volumes.md [hunk 6]
```diff
@@ -71,17 +71,17 @@ encrypted volume.
 
 Shared flag can be set while creating the encrypted volume using `--shared`
 It can also be enabled or disabled during run-time using `--shared on/off`.
-Volume must be in detached to state to enable shared flag on during run-time.
+Volume must be in detached to state to toggle shared flag during run-time.
```
Comment (piyush-nimbalkar) [inline manage/encrypted-volumes.md]: remove "to"
must be in detached state

File: manage/encrypted-volumes.md (hunk 4 above)
Comment (piyush-nimbalkar) [reply manage/encrypted-volumes.md]: you forgot to add space  after the # and before pxctl

File: manage/encrypted-volumes.md (hunk 5 above)
Comment (piyush-nimbalkar) [reply manage/encrypted-volumes.md]: The Replication Status word is still trimmed

**Review by piyush-nimbalkar — APPROVED** [review]
LGTM after nit picks

**Pull request merged**